### PR TITLE
Feat : Added landing page

### LINKED ISSUE #6: Update: landing page
Added links for categories in landing page.

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. landing page
QuizBeeLanding

### CODE REVIEW

**Review by Gautam2305 — COMMENTED** [review]
clean code, it's responsive too .... good to merge

File: css/main.css [hunk 1]
```diff
@@ -1,2 +1,9 @@
 @import url('./reset.css');
-@import url('./variables.css');
\ No newline at end of file
+@import url('./variables.css');
+
+
+@import url('./screens/landingPage.css');
+
+
+
```
Comment (Sameony) [inline css/main.css]: maybe, remove these blank lines

File: css/mediaQueries.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/* === media queries === */
+
+@media screen and (max-width: 870px){
```
Comment (Sameony) [inline css/mediaQueries.css:3]: media queries are more often than not, placed at the end of the file. so you can follow up on that

File: css/screens/landingPage.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+.text{
+    color: var(--text-color-light);
+    font-size: var(--normal-font-size);
+    margin: 1.9rem 0 2rem;
+    max-width: 550px;
+    line-height: 1.7;
+}
+/* .right-side{
+    padding: 0rem 1rem;
+} */
```
Comment (Sameony) [inline css/screens/landingPage.css]: do remove these unnecessary blocks of comments

File: index.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                            <span>e</span>
+                            <span>s</span>
+                        </div>
+                        <div class="category-contents">
+                            <ul>
+                                <li class="category-list">
+                                    <div class="left">
+                                        <i class="fas fa-globe-asia"></i>
+                                    </div>
+                                    <div class="right">
```
Comment (tanishq20) [inline index.html]: Class names can be more specific, instead of `left` and `right`.

**Review by Sameony — APPROVED** [review]
its good. Suggested a few structural adjustments.

File: css/screens/landingPage.css [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    color: var(--text-color) ;
+    margin-left: 4.5rem;
+    display: inline-block;
+    transition: .1s;
+}
+.links a:hover{
+    color: var(--primary-dark);
+    transform: scale(1.05);
+}
+.btn{
```
Comment (tanishq20) [inline css/screens/landingPage.css]: Generally `btn` in global class. So you can either move it to the utility folder or can change the name to more specific towards the section.

File: css/main.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/main.css]: sure..

**Review by tanishq20 — COMMENTED** [review]
Suggested small changes.

File: css/screens/landingPage.css (hunk 3 above)
Comment (RahulSingh9131) [reply css/screens/landingPage.css]: sure will remove that

File: css/mediaQueries.css (hunk 2 above)
Comment (RahulSingh9131) [reply css/mediaQueries.css:3]: yes i have created a separate css file for media queries, and imported that in the end of main.css

File: css/screens/landingPage.css (hunk 5 above)
Comment (RahulSingh9131) [reply css/screens/landingPage.css]: sure

**Review by Gautam2305 — APPROVED** [review]

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**